Source/UI/Common/Styles/StylingExtensions.swift

```diff
@@ -0,0 +1,73 @@
+//
```

Comment:
Applying of the style should be available to the integrator for keeping the same style of the buttons in onboarding flow.